terraform/aws-multi-cluster/aws_iam.tf

```diff
@@ -10,6 +10,12 @@ resource "aws_iam_role" "kubecost_federated_storage" {
   })
 
   tags = var.tags
+
+  lifecycle {
+    ignore_changes = [
+      role_last_used,
+    ]
+  }
```

Comment:
Thanks for adding this. Could you clarify why it's needed? If multiple users/roles touch the bucket, does it update the bucket's `role_last_used` attribute?

Reply:
Whenever I apply terraform, terraform was showing it as diff and re applying. I did not want to reapply for this particular diff